spec/components/schema/day_breakdowns.yaml

```diff
@@ -0,0 +1,294 @@
+      type: object
+      required:
+        - date_time
+        - vacancies
+      properties:
+        date_time:
+          type: string
+          format: date-time
+          description: Tour-option conduction date, it will be midnight of the conduction date if the activity uses opening hours.
+        vacancies:
+          type: integer
+          example: 10
+        unavailable_reason:
+          $ref: '#/components/schemas/UnavailableReason'
+        price_breakdown:
+          $ref: '#/components/schemas/PriceBreakdown'
+
+    PriceBreakdown:
+      type: object
+      required:
+        - customer_total
+      properties:
+        individuals:
+          type: array
+          items:
+            $ref: '#/components/schemas/IndividualCategoryPriceBreakdown'
+        groups:
+          type: array
+          items:
+            $ref: '#/components/schemas/GroupPriceBreakdown'
+        add-ons:
+          type: array
+          items:
+            $ref: '#/components/schemas/AddonPriceBreakdown'
+        deal_summary:
+          $ref: '#/components/schemas/DealSummary'
+        customer_total:
+            type: number
+            example: 40.0
+            description: The total price for which the tour option is sold to the customer.
+    UnavailableReason:
+      type: object
+      required:
+        - problem
+        - parameters
+      properties:
+        problem:
+          type: string
+          example: not_enough_participants
+        parameters:
+          type: object
+          additionalProperties:
+            type: string
+    MonetaryAmount:
+      type: object
+      description: Generic monetary amount object.
+      required:
+        - currency_iso_code
+        - amount
+      properties:
+        currency_iso_code:
+          type: string
+          description: ISO code of currency.
+          example: USD
+        amount:
+          type: number
+          format: double
+          description: Amount in currency.
+          example: 19.95
+    Price:
+      type: object
+      required:
+        - retail_price
+      properties:
+        customer_base_price:
+          type: number
+          example: 45.0
+          description: The price for which the tour option category is sold to the customer excluding discounts if any.
+        retail_price:
+          type: number
+          example: 40.0
+          description: The price for which the tour option category is sold to the customer.
+    IntegerRange:
+      type: object
+      properties:
+        min:
+          type: integer
+        max:
+          type: integer
+    PriceBreakdownBase:
+      type: object
+      required:
+        - quantity
+        - price
+      properties:
+        quantity:
+          type: integer
+          description: Number of individual/group participants, or addon items.
+          example: 5
+        price:
+          $ref: '#/components/schemas/Price'
+    IndividualCategoryPriceBreakdown:
+      allOf:
```

Comment:
This `allOf` doesn't always work so well in OpenAPI generator. You might want to remove the use of `allOf` and instead just duplicate the properties from `PriceBreakdownBase` where needed, especially since this will be a publicly exposed spec.  We will be removing the `allOf` from our spec. The only benefit is that it saves some typing in the Yaml file.

Reply:
I'll apply this suggestion, thanks Hinrik!